lib/transformer.dart

```diff
@@ -258,21 +256,23 @@ class _BootstrapFileBuilder {
               '(possibly transitive).');
         }
       }
+
       readSuperClassAnnotations(clazz.supertype);
       _readAnnotations(clazz);
     }
   }
 
   bool _readAnnotations(Element element) {
     var found = false;
-    if (element.metadata.isEmpty) return found;
+    // analyzer 0.29 doesn't allow this optimization :
+    //if (element.metadata.isEmpty) return found;
```

Comment:
Is there a test or anything which fails based on this? I would like to send something to the analyzer folks so they can look at it.